api/v1alpha1/database_webhook.go

```diff
@@ -106,6 +106,11 @@ func instanceSpecDefaulterForCreate(instance *Instance) {
 		instance.TMInfo.QuarterlySnapshotMonth = "Jan"
 	}
 
+	// additional arguments defaulting logic
+	if instance.AdditionalArguments == nil {
+		databaselog.Info("Initialzing empty TypeDetails...")
```

Comment:
I'd suggest a more intuitive log msg: "Initializing additional db arguments"
also typo in 'Initialzing'